docker-compose.yml

```
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  groupware-whitelabel-postgres:
+    image: postgres:14
+    container_name: '${APP_NAME}_POSTGRES'
+    environment:
+      POSTGRES_USER: ${POSTGRES_USERNAME}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DATABASE}
+      PGDATA: /var/lib/postgresql/data
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+    networks:
+      - groupware-whitelabel-network
+
+  groupware-whitelabel-app:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    container_name: '${APP_NAME}_APP'
+    image: app-groupware-whitelabel
+    volumes:
+      - .:/app
+      - /app/node_modules
+    env_file:
+      - .env
+    environment:
+      POSTGRES_HOST: groupware-whitelabel-postgres
+    ports:
+      - ${APP_PORT}:${APP_PORT}
```

Ini langsung aja kang ikutin yang ada di dockerfile nya, karena semua image port nya sudah dibuat hardcode
```suggestion
      - 3000:3000
```